app/uk/gov/hmrc/gform/gform/SummaryController.scala

```diff
@@ -109,15 +111,16 @@ class SummaryController(
                      redirectToSummary.pure[Future]
                    )
         } yield result
-
+        val formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss'Z'")
+        val envelopeExpiryDate = cache.form.envelopeExpiryDate.fold("SomeString")(_.ldt.format(formatter))
```

Comment:
What is this SomeString about?

Reply:
This was a filler string that I forgot to change. Now that .fold is done once in the view the .fold in this class is no longer required and has been removed.